drivers/amplifiers/ada4250/ada4250.c

```diff
@@ -148,11 +148,6 @@ int32_t ada4250_update_desc(struct ada4250_dev *dev)
 		return ret;
 	dev->refbuf_en = reg_data & ADA4250_REFBUF_MSK;
 
-	ret = ada4250_read(dev, ADA4250_REG_SNSR_CAL_VAL, &reg_data);
-	if (ret != 0)
-		return ret;
-	dev->offset_uv = reg_data & ADA4250_SNSR_CAL_VAL_MSK;
```

Comment:
Is this not needed anymore?

Reply:
I don't think so. The only way the dev->offset_nv is going to be updated is through the ada4250_set_offset() function. Or else we need to maintain a sperate function for just calculating the offset in nanovolts.